packages/eslint-config-sds/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "author": {
-    "name": "SICK Integrated Automation - UX and UI Services Team",
+    "name": "SICK Global UX Foundation",
```

Comment:
sollen wir nicht einfach SICK AG reinschreiben?

Reply:
I think it would be beneficial to at least name the group of persons behind this. Something like a more global CODEOWNSERS file that makes it clear that there is a responsible person for this project. Should be discussed.